client/components/status/status.js

```diff
@@ -405,8 +409,11 @@ export default class Status extends React.Component {
       if (options.showRadiusUsage) {
         options.userChecks = response.data.checks;
         options.userChecks.forEach((check) => {
-          if (check.value === String(check.result)) {
+          if (Number(check.value) <= check.result) {
             isPlanExhausted = true;
+            if (settings.subscriptions && !warningMessage) {
```

Comment:
The usage limit is not strictly a feature of openwisp subscriptions right? Only plans and payments.